example-cluster/images/xenial

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:xenial-20180123
+FROM ubuntu:bionic
```

Comment:
Hmm perhaps this file should be renamed `s/xenial/bionic/` as well. or can we just remove this example image?

Reply:
:lol: didn't even notice the name!

I think we can probably remove this and the rest of the example cluster stuff (especially since we don't use it) unless we rework it to work internally (after things like our pypi blocks), but I'll leave that for future luisp to do :)